src/Raven.Server/Smuggler/Documents/DatabaseSource.cs

```diff
@@ -106,7 +106,28 @@ public DatabaseItemType GetNextType()
 
         public DatabaseRecord GetDatabaseRecord()
         {
-            return _database.ReadDatabaseRecord();
+            var databaseRecord = _database.ReadDatabaseRecord();
+
+            // filter server wide backup tasks;
+            var backupTasksToRemove = new List<int>();
+            for (var i = 0; i < databaseRecord.PeriodicBackups.Count; i++)
+            {
+                var periodicBackup = databaseRecord.PeriodicBackups[i];
+                if (periodicBackup.Name == null)
+                    continue;
+
+                if (periodicBackup.Name.StartsWith(ServerWideBackupConfiguration.NamePrefix, StringComparison.OrdinalIgnoreCase))
+                {
+                    backupTasksToRemove.Add(i);
+                }
+            }
+
+            foreach (var toRemove in backupTasksToRemove)
+            {
+                databaseRecord.PeriodicBackups.RemoveAt(toRemove);
```

Comment:
This can throw IndexOutOfRange, right?

Reply:
Yes, it can. Better to simply remove them in the look above and then `i--`

Reply:
done